stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,929 @@
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ullam possimus corrupti architecto perspiciatis, 
+            delectus necessitatibus incidunt numquam asperiores tenetur iure. 
+            Cum consequuntur impedit repellendus esse, facere autem optio consequatur nobis?
+        </div>
+        <div class="fd-dynamic-page__header-visibility-container fd-dynamic-page__header-visibility-container--no-shadow" aria-hidden="true">
+            <div class="fd-dynamic-page__header-visibility-container--left-gradient"></div>
+            <div class="fd-dynamic-page__header-visibility-container--button-group">
+                <button class="fd-dynamic-page__collapse-button fd-button fd-button--compact" 
+                        aria-label="See More" 
+                        aria-expanded="false" 
+                        aria-haspopup="true" 
+                        aria-controls="fddplhvcm2">
+                        <i class="sap-icon--slim-arrow-down"></i>
+                </button>
+            </div>
+            <div class="fd-dynamic-page__header-visibility-container--right-gradient"></div>
+        </div>
+        <div class="fd-dynamic-page__tabs--overflow">
+            <ul class="fd-tabs fd-tabs--l fd-dynamic-page__tabs fd-dynamic-page__tabs--s fd-dynamic-page__tabs--add-shadow" role="tablist">
+                <li role="listitem" class="fd-tabs__item">
+                    <a class="fd-tabs__link" aria-controls="fuCwV554" href="#fuCwV554" role="tab">
+                        <span class="fd-tabs__tag">
+                            Link
+                        </span>
+                    </a>
+                </li>
+                <li role="listitem" class="fd-tabs__item">
+                    <a class="fd-tabs__link" aria-controls="AiWfz169" aria-selected="true" href="#AiWfz169" role="tab">
+                        <span class="fd-tabs__tag">
+                            Selected
+                        </span>
+                    </a>
+                </li>
+            </ul>  
+        </div>     
+        <div class="fd-dynamic-page__content fd-dynamic-page__content--s">
+            <div class="fd-tabs__panel" aria-expanded="false" id="fuCwV554" role="tabpanel">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam 
+                eum tempore aliquid dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam eum tempore aliquid 
+                dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+            </div>
+            <div class="fd-tabs__panel" aria-expanded="true" id="AiWfz169" role="tabpanel">
+                <h4>This is an example of a collapsed title area and header and instead of displaying a Summary Line</h4>
```

Comment:
This was a bit confusing to me, maybe reword it to:

This is an example of a summary line, without the expand/collapse feature.

Reply:
modified